.github/workflows/release.yaml

```diff
@@ -0,0 +1,36 @@
+name: goreleaser
+
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
are we going to release all commits ?

Reply:
copied from the image flow, I can limit it to just tags